scripts/CSSRE/rolling-restart-brokers/README.md

```diff
@@ -0,0 +1,32 @@
+# get-kafka-topics
+
+This scripts runs the `rolling-restart-brokers.sh` script to manually roll the Kafka brokers.
+
+## Usage
+
+### Running locally
+
+```bash
+Usage: ./rolling-restart-brokers.sh -n KAFKA_NAMESPACE [-s STATEFUL_SET | -p POD]
+
+# Examples
+./rolling-restart-brokers.sh -n kafka-ID
```

Comment:
Just an minor comment here: `Kafka-ID` is used here in place of `KAFKA_NAMESPACE`. Might be best to change to avoid confusion this as kafka ID and kafka namespace are not the same.
`KAFKA_NAMESPACE=kafka-$KAFKA_ID`

Reply:
Changed as suggested.